tests/integration/examples.py

```diff
@@ -44,6 +52,8 @@ def it_does_start(self, in_example_dir):
         assert os.path.isfile('output')
         assert open('output').read() == test_string
 
+        stop_service_check('tail')
```

Comment:
This doesn't belong in the test.
It should be in the cleanup of in_example_dir.